assets/src/edit-story/elements/text/display.js

```diff
@@ -43,6 +43,7 @@ import {
 import createSolid from '../../utils/createSolid';
 import stripHTML from '../../utils/stripHTML';
 import { isOutsideBorder } from '../../components/elementBorder/utils';
+import useColorTransformHandler from '../shared/useColorTransformHandler';
```

Comment:
What about the `TextEdit` component? This is only for non-editing text, but you also need to be able to do this color changing while in edit-mode.

Perhaps both this and the previous issue can be solved by similarly using a CSS variable for text color as you do for border color? Then you'd just have to use that CSS variable in the rule set for [the faux selection styling here](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/components/richText/fauxSelection.js#L125) - as well as for the entire text field if in display-mode. And then you might even be able to generalize the CSS variable transform handler as well as it will be used 3 times then.

At least I think, that would work?

Reply:
Added in 248c4b8

Reply:
A few issues, but nice refactor.